### PR TITLE
Enterprise 1.9: document node labels

### LINKED ISSUE #2587: InfluxDB Enterprise - user defined node labels
New feature for InfluxDB Enterprise 1.9.

See influxdata/plutonium#3343

### PR DESCRIPTION
**Pull request by pierwill**
Closes #2587.

- [x] Tests pass (no build errors)
- [x] Rebased/mergeable

### CODE REVIEW

File: content/enterprise_influxdb/v1.9/administration/cluster-commands.md [hunk 1]
```diff
@@ -664,6 +665,107 @@ Successfully left cluster
...
+2       localhost:8291  1.9.x-c1.9.x  {}
+3       localhost:8391  1.9.x-c1.9.x  {}
+```
+
+##### Update, delete, and show labels
+
+Update an existing label on one node, delete a label on the other node, and show the cluster:
+
+```sh
+$ ./influxd-ctl node-labels set -nodeid 4 -labels '{"hello":"world"}'
```
Comment (kelseiv) [inline content/enterprise_influxdb/v1.9/administration/cluster-commands.md:730]:
```suggestion
$ ./influxd-ctl node-labels update -nodeid 4 -labels '{"hello":"world"}'
```

Comment (kelseiv) [reply content/enterprise_influxdb/v1.9/administration/cluster-commands.md:730]: Does both `set` and `update` work to update a label?

File: content/enterprise_influxdb/v1.9/administration/cluster-commands.md [hunk 2]
```diff
@@ -664,6 +665,107 @@ Successfully left cluster
...
 
+### `node-labels`
+
+Assign user-defined labels to meta and data nodes in a cluster.
+Labels are JSON key-value pairs.
+
+#### Syntax
+
+```bash
+influxd-ctl [options] node-labels <command> [options] [<args>]
```
Comment (kelseiv) [inline content/enterprise_influxdb/v1.9/administration/cluster-commands.md:676]: Do we document the node-labels [options] somewhere? 
Didn't see `-nodeid` or `-labels` other than in examples...do we have other options?

Comment (pierwill) [reply content/enterprise_influxdb/v1.9/administration/cluster-commands.md]: Looks like these are the only two.

File: content/enterprise_influxdb/v1.9/tools/influxd-ctl.md [hunk 3]
```diff
@@ -653,6 +660,111 @@ Successfully left cluster
...
+```bash
+influxd-ctl [options] node-labels <command> [options] [<args>]
+```
+
+Available commands are:
+
+- `set`: set (add or update) one or more labels on a node
+- `delete`: delete one or more labels from a node
+
+Required options are when operating on labels:
```
Comment (pierwill) [inline content/enterprise_influxdb/v1.9/tools/influxd-ctl.md:679]:
```suggestion
Required options when operating on labels:
```

**Pull request merged**